llvm/bindings/ocaml/debuginfo/llvm_debuginfo.ml

```diff
@@ -592,26 +592,6 @@ external dibuild_create_parameter_variable :
   Llvm.llmetadata
   = "llvm_dibuild_create_parameter_variable_bytecode" "llvm_dibuild_create_parameter_variable_native"
 
-external dibuild_insert_declare_before :
-  lldibuilder ->
-  storage:Llvm.llvalue ->
-  var_info:Llvm.llmetadata ->
-  expr:Llvm.llmetadata ->
-  location:Llvm.llmetadata ->
-  instr:Llvm.llvalue ->
-  Llvm.lldbgrecord
-  = "llvm_dibuild_insert_declare_before_bytecode" "llvm_dibuild_insert_declare_before_native"
-
-external dibuild_insert_declare_at_end :
-  lldibuilder ->
-  storage:Llvm.llvalue ->
-  var_info:Llvm.llmetadata ->
-  expr:Llvm.llmetadata ->
-  location:Llvm.llmetadata ->
-  block:Llvm.llbasicblock ->
-  Llvm.lldbgrecord
-  = "llvm_dibuild_insert_declare_at_end_bytecode" "llvm_dibuild_insert_declare_at_end_native"
-
```

Comment:
See comments above from debuginfo_ocml.c - I don't these should be deleted

Reply:
Restoring the deleted functions.